### PR TITLE
Upgrade projects

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**
Howdy folks,

All projects have been upgraded to target .NET Core 3.1. I have taken the following steps:

1. Run in Rider on macOS and VS 2019 ReSharper on Windows
    1. All projects build in both environments except **Xaml** because it targets WPF.
    1. WPF is running on .NET Core 3.1
1. MVC -> WebApplication (ported to ASP.NET Core)
1. Clipboard works cross-platform now thanks to [TextCopy](https://www.nuget.org/packages/TextCopy) and @simoncropp
1. F# project now targets .NET Core 3.1

## Notes

- I did not upgrade packages to the latest versions
- `AssemblyInfo` was unnecessary. If we want to update the `.csproj` files with copyright we can later.
- `System.Windows.Forms` has a `shim` for now, but we should talk about this project and think about a cross-platform approach.
- In general, a **X-Plat** friendly guide is preferred to something that steers folks towards one OS.

### CODE REVIEW

File: 05-LanguagesAndFrameworks/WebApplication/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</header>
+<div class="container">
+    <main role="main" class="pb-3">
+        @RenderBody()
+    </main>
+</div>
+
+<footer class="border-top footer text-muted">
+    <div class="container">
+        &copy; 2020 - WebApplication - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
```
Comment (maartenba) [inline 05-LanguagesAndFrameworks/WebApplication/Views/Shared/_Layout.cshtml:40]:
```suggestion
        &copy; @System.DateTime.UtcNow.Year - WebApplication - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
```

**Review by maartenba — CHANGES_REQUESTED** [review]
* Minor nit in MVC project footer (helps keep things "up to date" by showing current year
* I'd go with updating packages where possible as well, but does not necessarily have to be part of this PR

File: 05-LanguagesAndFrameworks/WebApplication/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (khalidabuhakmeh) [reply 05-LanguagesAndFrameworks/WebApplication/Views/Shared/_Layout.cshtml:40]: Take it up with Microsoft :P

**Comment by khalidabuhakmeh** [comment]
I'll go through and update the packages in a different PR.

**Review by maartenba — APPROVED** [review]
OK for me

**Comment by khalidabuhakmeh** [comment]
I upgraded the packages as well, there weren't that many of them.

**Review by maartenba — APPROVED** [review]

**Pull request merged**